src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.props

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
+    <BuildToolsVersioningPropsImported>true</BuildToolsVersioningPropsImported>
```

Comment:
I think internal only properties are generally prefixed by '_' in BuildTools, but it's certainly not consistent throughout.

Reply:
I was kind of thinking of this as public, and some external repo could use this to check whether it was already imported, and not import it again....  thoughts?

Reply:
Makes sense

Reply:
Similar to this

https://github.com/Microsoft/msbuild/blob/f72bd2eaa37476819ab7647f63d445df8055dab6/src/Tasks/Microsoft.Common.props#L140

Maybe I should rename it to the same name format?